Pharus/Scenes/Student Project Detail/StudentProjectDetailView.swift

```diff
@@ -14,6 +14,8 @@ protocol StudentProjectDetailViewDelegate: AnyObject {
     func taskCheckboxTapped(task: TaskModel)
 }
 
+//  swiftlint: disable type_body_length file_length
```

Comment:
O padrão mais utilizado para as notações do swiftlint é assim:

```suggestion
// swiftlint:disable type_body_length file_length
```